tools/nl/embeddings/csv/palm_alternatives.csv

```diff
@@ -0,0 +1,1246 @@
+Id,PaLM_Generated_Alternatives
+AmountFarmInventory_WinterWheatForGrain,
+Amount_FarmInventory_BarleyForGrain,
+Amount_FarmInventory_CornForSilageOrGreenchop,
+Amount_FarmInventory_Cotton,
+Amount_FarmInventory_DryEdibleBeans,
+Amount_FarmInventory_DurumWheatForGrain,
+Amount_FarmInventory_Forage,
+Amount_FarmInventory_OatsForGrain,
+Amount_FarmInventory_OtherSpringWheatForGrain,
```

Comment:
Should we only keep SVs with alternatives?

Reply:
I thought about it but I think this makes it easy to also identify the failures to generate alternatives?

Reply:
Now being used as an input.